Localization Italian - Updates and additions

Harvester62 opened the pull request:
Updated localization with the latest version of these Scripts/Extensions:

animator v6
StylePile
Alpha Canvas

Changed the description for the script "To Infinity and Beyond" (Verso l'infinito e oltre) from 'n' to 'Esegui n volte' (Run n times) Added localization for the 'Random' and 'Random Grid' scripts I changed both the title and the parameters Steps and CFG labels to make it clearer that the min/max order can be inverted. Added a few missing translations and character's corrections here and there. Still a work in progress but I will slowly fix wrong thing when I find them.

I have to get the habit to edit the file in the forked repo instead of the one inside my local installation, I know, forgive me.

Harvester62 committed:
Localization Italian - Updates and additions

Updated localization with the latest version of these Scripts/Extensions:

animator v6
StylePile
Alpha Canvas

Changed the description for the script "To Infinity and Beyond" (Verso l'infinito e oltre) from 'n' to 'Esegui n volte' (Run n times)
Added localization for the 'Random' and 'Random Grid' scripts I changed both the title and the parameters Steps and CFG labels to make it clearer that the min/max order can be inverted.
Added a few missing translations and character's corrections here and there. Still a work in progress but I will slowly fix wrong thing when I find them.

Harvester62 committed:
Merge branch 'AUTOMATIC1111:master' into Italian

Harvester62 committed:
Italian localization - Updated a few terms

I've updated a few terms and descriptions. Waiting for review and, hopefully, approval.

Harvester62 committed:
Merge branch 'AUTOMATIC1111:master' into Italian

Harvester62 committed:
Merge branch 'AUTOMATIC1111:master' into Italian

Harvester62 committed:
Italian localization - Updates Extensions

Added translations of the new Extensions tab, and a few corrections to some previously translated descriptions/terms.

EugenioBuffo reviewed (commented):
Hi, is this file here purely by chance or was it purposely included outside the "localizations" folder?
Does it contain necessary changes or can it be removed as it is not part of the folder?

Harvester62 committed:
Merge branch 'AUTOMATIC1111:master' into Italian

EugenioBuffo committed:
Deleted extra it_IT localization

Deleted it_IT localization file outside localization folder as it was not intentionally committed

EugenioBuffo committed:
Removed duplicated string from it_IT localization

Removed "Peso dei bordi del punto focale" duplicated string from "Focal point edges weight"

EugenioBuffo commented:
Hello,

you're doing a great job with the Italian localization file!

For the editor: I have used both Atom and VSCode and personally I prefer VSCode more for multiple reasons.

With the GIT extension it helps a lot to have a complete view of the current changes, commits and state prior to the change.

It also has a great interface that allows you to diff with a lot of ease and clarity.

Thank You!
